### PR TITLE
Support for servicecomb-kie.

### PR DESCRIPTION
**Pull request by surechen**
Signed-off-by: surechen <chenshuo17@huawei.com>

### CODE REVIEW

File: servicecombkie/kie_client.go [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+		EnableSSL:     kieClient.EnableSSL,
+		AutoDiscovery: kieClient.AutoDiscovery,
+		Namespace:     kieClient.Namespace,
+		TenantName:    kieClient.TenantName,
+	}
+	return optionInfo
+}
+
+//InitConfigKie initialize the Kie Client
+func InitConfigKie(options config.Options) (config.Client, error) {
```
Comment (tianxiaoliang) [inline servicecombkie/kie_client.go]: merge InitConfigKie  and NewKieClient into one method "NewClient"

**Review by tianxiaoliang — CHANGES_REQUESTED** [review]

File: servicecombkie/kie_client.go [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+func (kieClient *Client) PullConfigs(serviceName, version, app, env string) (map[string]interface{}, error) {
+	openlogging.GetLogger().Debugf("KieClient begin PullConfigs")
+	labels := map[string]string{"servicename": serviceName, "version": version, "app": app, "env": env}
+	configsInfo := make(map[string]interface{})
+	configurationsValue, err := kieClient.KieClient.SearchByLabels(context.TODO(), sckieclient.WithGetProject(serviceName), sckieclient.WithLabels(labels))
+	if err != nil {
+		openlogging.GetLogger().Errorf("Error in Querying the Response from Kie %s %s %s %s %s", err.Error(), serviceName, version, app, env)
+		return nil, err
+	}
+	openlogging.GetLogger().Debugf("KieClient begin PullConfigs1")
```
Comment (tianxiaoliang) [inline servicecombkie/kie_client.go]: use structured log "WithTag", you can search example in go chassis

**Review by tianxiaoliang — CHANGES_REQUESTED** [review]

File: servicecombkie/kie_client.go [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+	}
+	openlogging.GetLogger().Debugf("KieClient Initialized successfully")
+}
+
+// PullConfigs is used for pull config from servicecomb-kie
+func (kieClient *Client) PullConfigs(serviceName, version, app, env string) (map[string]interface{}, error) {
+	openlogging.GetLogger().Debugf("KieClient begin PullConfigs")
+	labels := map[string]string{"servicename": serviceName, "version": version, "app": app, "env": env}
+	configsInfo := make(map[string]interface{})
+	configurationsValue, err := kieClient.KieClient.SearchByLabels(context.TODO(), sckieclient.WithGetProject(serviceName), sckieclient.WithLabels(labels))
```
Comment (tianxiaoliang) [inline servicecombkie/kie_client.go]: WithGetProject to "WithProject"

Comment (tianxiaoliang) [reply servicecombkie/kie_client.go]: project should be a options attribute and set by NewClient, by default is "dafault"

File: servicecombkie/kie_client.go (hunk 2 above)
Comment (surechen) [reply servicecombkie/kie_client.go]: Edit

File: servicecombkie/kie_client.go (hunk 1 above)
Comment (surechen) [reply servicecombkie/kie_client.go]: Edit

**Review by tianxiaoliang — APPROVED** [review]

**Pull request merged**